### ISSUE TITLE
We need google or bing

### ISSUE THREAD

**Issue by jdd-software** [SOBotics/Guttenberg#105]
Today again we found a plag. user, Felix have done great work with algorithms to find possibile plag, but the `/related` search really sucks.

The solution would be to find a good strategy and use google or bing api, however the problem is the api-quota.

So this is my idea:

1. We develop a software that can run on cliente macchine (hence multiple people can run it)
2. On SOBotics.org we develop a service, where the cliente macchine can registered there current IP.
3. Guttenberg forwards it search request to SOBotics.org, the application distribution the call to different clients registered that responds.
4. SOBotics.org responds with possibile plag. id
5. Guttenberg procede as usual

Basically this way we can use multiple ip and api-key to query google or bing (or both), note this system could also be use to query SE-Api in the future and let us run all bots un SOBotics.org, using our own pc only as clients to distribute ip/api.-key.

**Comment by ArtOfCode-** [comment]
This sounds complicated, and probably also runs afoul of search API TOS. I'd be surprised if Google's API TOS didn't categorise using multiple clients from multiple IPs to work around volume restrictions as abuse.
